fixtures/emails/n1_api_call.txt

```diff
@@ -16,7 +16,7 @@ Tags
 
 * browser = Chrome 108.0.0
 * browser.name = Chrome
-* client_os = Mac OS X 10.15.7
+* client_os = Mac OS X &gt;=10.15.7
```

Comment:
Is this change related to the dependency bump?

Reply:
Yes, this PR specifically: https://github.com/getsentry/relay/pull/1945